### PR TITLE
CAMEL-17938 Add tests in camel-aws2-cw-starter

### PR DESCRIPTION
**Pull request by JiriOndrusek**
Issue: https://issues.apache.org/jira/browse/CAMEL-17938

### CODE REVIEW

File: components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/BaseCw.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.aws2.cw;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.Configuration;
```
Comment (cunningt) [inline components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/BaseCw.java:20]: Needs to be org.springframework.context.annotation.Configuration

File: components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/CwComponentTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.aws2.cw;
+
+import org.apache.camel.Configuration;
```
Comment (cunningt) [inline components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/CwComponentTest.java:19]: Needs to be org.springframework.context.annotation.Configuration

**Review by cunningt — CHANGES_REQUESTED** [review]

File: components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/BaseCw.java (hunk 1 above)
Comment (JiriOndrusek) [reply components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/BaseCw.java:20]: Thanks for spotting it, I'll fix it

Comment (JiriOndrusek) [reply components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/BaseCw.java]: done

File: components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/CwComponentTest.java (hunk 2 above)
Comment (JiriOndrusek) [reply components-starter/camel-aws2-cw-starter/src/test/java/org/apache/camel/component/aws2/cw/CwComponentTest.java]: done

**Pull request merged**